packages/web/src/components/notification/Notifications/FollowNotification.tsx

```diff
@@ -1,7 +1,21 @@
-import React from 'react'
+import React, { useCallback } from 'react'
 
+import { push } from 'connected-react-router'
+import { useDispatch, useSelector } from 'react-redux'
+
+import { ID } from 'common/models/Identifiers'
+import { getUserId } from 'common/store/account/selectors'
 import { Follow } from 'common/store/notifications/types'
 import { formatCount } from 'common/utils/formatUtil'
+import {
+  setUsers,
+  setVisibility
```

Comment:
nit: Might be useful to rename these imports so it's clear they're from the userListModal? `setUserListModalVisibility` etc seems verbose though so up to you

Reply:
100%